guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc

```diff
@@ -13,9 +13,16 @@ You can use your browser to access the VNC console of VMs created by {Project}.
 .Prerequisites
 * You must have a virtual machine created by {Project}.
 * For existing virtual machines, ensure that the *Display type* in the *Compute Resource* settings is *VNC*.
+ifdef::katello,orcharhino,satellite[]
 * You must import the Katello root CA certificate into your {ProjectServer}.
 Adding a security exception in the browser is not enough for using noVNC.
 For more information, see {ConfiguringUserAuthenticationDocURL}importing-the-katello-root-ca-certificate[Installing the Katello root CA certificate] in _{ConfiguringUserAuthenticationDocTitle}_.
+endif::[]
+ifdef::foreman-deb,foreman-el[]
+* Import the CA certificate that is set as `websockets_ssl_cert`.
```

Comment:
The `websockets_ssl_cert` value is typically a certificate (which is signed by a Certificate Authority). You really want to import the certificate that signed it to avoid having to do this procedure again on all browsers when you renew your certificate. Of course, you still need to do that when you chance CA, but that's less common.

How about keeping this very generic for now and consider that good enough?

```suggestion
* You have imported the CA certificate used for {Project} into your browser.
```

I also consider that a better sentence for the Katello (& friends) cases because you don't import it into your `{ProjectServer}` but really into your browser.

Reply:
Yes, I agree. -> Applied and simplified.